pipeline/backend/docker/convert.go

```diff
@@ -82,7 +82,10 @@ func toHostConfig(proc *backend.Step) *container.HostConfig {
 			config.Tmpfs[path] = ""
 			continue
 		}
-		parts := strings.Split(path, ":")
+		parts := strings.SplitN(path, ":", 1)
```

Comment:
Why do you limit the split to 1? Shouldn't that be 2?

Reply:
because windows volume need additional ":" for drive likes this.
~~~
        "pipeline_default:c:\test:rw"
~~~